src/ray/core_worker/transport/task_receiver.h

```diff
@@ -96,9 +96,11 @@ class TaskReceiver {
 
   bool CancelQueuedNormalTask(TaskID task_id);
 
-  /// Cancel an actor task queued in the actor scheduling queue for caller_worker_id.
-  /// Return true if a task is queued or executing. False otherwise.
-  /// If task is not executed yet, this will guarantee the task won't be executed.
+  /// Cancel an actor task that is queued for execution, but hasn't started executing yet.
+  ///
+  /// Returns a boolean indicating if the task is present in the executor at all. If not,
```

Comment:
nit: Mention “true”/“false” explicitly. It makes it easier to read, though this may just be due to my English ability. Feel free to ignore if this is easy to understand for most readers.

```suggestion
  /// Returns true indicating if the task is present in the executor at all. If not,
```

Reply:
you are part of the intended audience so this is really helpful feedback :)